### PR TITLE
Feature addition: issue delete

### PR DESCRIPTION
**Pull request by ggrpmnn**
I saw the Issue.Delete issue open (#60) and took a stab at adding the code. My first commit fixes several minor issues that atom's `gometalinter` plugin discovered, and the next commit adds the delete functionality and unit test. Let me know if anything can be improved or needs to be changed.

### CODE REVIEW

File: metaissue.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ type MetaIssueType struct {
 	IconUrl     string                `json:"iconurl,omitempty"`
 	Name        string                `json:"name,omitempty"`
 	Subtasks    bool                  `json:"subtask,omitempty"`
-	Expand      string                `json:"expand,omitempty"`
+	Expand      string                `json:",omitempty"`
```
Comment (andygrunwald) [inline metaissue.go]: Why this was removed?

**Comment by andygrunwald** [comment]
Thanks for this PR. It looks good. Only the one question in the metaissue.go is open

File: metaissue.go (hunk 1 above)
Comment (ggrpmnn) [inline metaissue.go]: It was causing a `go vet` issue because both Self and Expand had the JSON expand key.

**Comment by andygrunwald** [comment]
Thank you :)

**Pull request merged**